src/metabase/driver/postgres.clj

```diff
@@ -287,28 +287,32 @@
 (def ^:private ssl-params
   "Params to include in the JDBC connection spec for an SSL connection."
   {:ssl        true
-   :sslmode    "require"
-   :sslfactory "org.postgresql.ssl.NonValidatingFactory"})
+   :sslmode    "require"})
```

Comment:
is this going to stop working for anybody that previously relied on the fact that `sslfactory=org.postgresql.ssl.NonValidatingFactory` was getting merged in there? E.g. if the JVM doesn't have their root certificate. Will things stop working after the upgrade until they manually go add this in to the connection URL?